packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [23.0.0]
+### Added
+- Add `onBreak` handler to `CodefiTokenPricesServiceV2` ([#3677](https://github.com/MetaMask/core/pull/3677))
+  - This allows listening for "circuit breaks", which can indicate an outage. Useful for metrics.
+- Add `fetchTokenContractExchangeRates` utility method ([#3657](https://github.com/MetaMask/core/pull/3657))
+
+### Changed
+- **BREAKING:** Bump `@metamask/approval-controller` dependency and peer dependency from `^5.1.0` to `^5.1.1` ([#3695](https://github.com/MetaMask/core/pull/3695))
```

Comment:
To continue the discussion on peer dependencies, why is this breaking if it's just a patch bump? I know it's defined as a peer dependency, but shouldn't we expect changes in 5.1.1 to be compatible with a 5.1.0-version controller?

Reply:
The changes should be compatible, but this forces anyone on v5.1.0 to upgrade. The fact that it's just a patch bump hopefully means that this is a breaking change that is very easy to accommodate.

Reply:
Okay. But if these changes are compatible, why mark them as breaking?

Reply:
The change in the peer dependency is a breaking change, making this package incompatible with projects using v5.1.0 of the approval controller.

v5.1.0 and v5.1.1 should be compatible with each other, in theory. But this package is not compatible with both of them. That's what this change in the peer dependency range is communicating.